.github/workflows/func_tests.yaml

```diff
@@ -0,0 +1,25 @@
+name: Functionality validation
+
+on:
+  pull_request_review:
+    types: [ submitted ]
+    branches: [ main ]
+
+jobs:
+  func:
+    name: Functional tests
+    runs-on: ubuntu-latest
+    if: >-
+      github.event.review.state == "approved"
+      || github.event.review.body == "recheck"
```

Comment:
These double-quotation marks need to change to singles, otherwise the job will result in `Unexpected symbol` error. 

Ref: https://github.com/actions/runner/issues/866

Reply:
ack